build.gradle

```diff
@@ -362,6 +364,18 @@ asciidoctor {
   inputs.dir file("spock-spring/boot2-test/src/test/groovy/org/spockframework/boot2")
   inputs.dir file("spock-spring/spring3-test/src/test/groovy/org/spockframework/spring3")
   inputs.dir file("spock-spring/spring5-test/src/test/groovy/org/spockframework/spring5")
+
+  doFirst {
+    def errors = []
+    sourceDir.eachFileRecurse(FILES) {
+      if ((it.name =~ /(?<!^index)\.adoc$/) && (it.text =~ /<<[^>#]+\.adoc#,[^>]+>>/)) {
```

Comment:
This only finds a subset, it doesn't really check whether the id is present, should we use a more powerful solution, e.g., https://github.com/aim42/htmlSanityCheck instead?

Reply:
I now also updated the Asciidoctor version.
With the latest version it at least warns about the wrong id.
Do you think this is enough?